desktop/renderer-app/src/constants/error-code.ts

```diff
@@ -77,7 +77,6 @@ export const RequestErrorMessage = {
 
     [RequestErrorCode.RecordNotFound]: "replay-does-not-exist",
 
-    // TODO: translate messages below
```

Comment:
@hyrious Is this comment redundant?

Reply:
These i18n keys below ware not translated when I wrote this comment. You can check out whether they are added to zh-CN.json to know if it can be deleted.

Reply:
No. They are not translated

Reply:
Can this change be withdrawn? @GuYith

Reply:
Yes, I'm sorry about this change.